### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by jdidion**
* Fix badge links
* Remove text about syntax strictness that pertains to v1.x and replace with test about wdlTools strictness
* Add limitations section
* Mention CWL support is under development

### CODE REVIEW

**Review by mhrvol — DISMISSED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -78,71 +66,51 @@ task count_bam {
...
-    }
-    command {
-        ls -lh ~{ref}
-    }
-}
-```
+## Limitations
 
+* Calls with missing arguments have limited support.
+* The new `Directory` type in WDL development/2.0 is not yet supported
```
Comment (commandlinegirl) [inline README.md:97]:
```suggestion
* The new `Directory` type in WDL development/2.0 is not yet supported.
```

File: README.md [hunk 2]
```diff
@@ -78,71 +66,51 @@ task count_bam {
...
+The compiled workflow can be executed via the DNAnexus command line client or web UI:
+
 ![this](doc/bam_chrom_counter.png)
 
 ## Strict syntax
 
-One of the compiler phases takes a workflow apart, and extracts standalone tasks and sub-workflows. This requires a lexical analysis on the WDL program. It currently uses a simple regular expression to detect task/workflow start and end. This means that a task has to adhere to the following rules:
-1. no extra text is allows after the final closing bracket
-2. within the task body, closing brackets may not start at the beginning of a line.
+dxCompiler uses [wdlTools](https://github.com/dnanexus-rnd/wdlTools), a parser that adheres strictly to the specifications. Most of the problematic automatic type conversions that are allowed by some other WDL runtime engines are not allowed by dxCompiler. Please use the command line tools in wdlTools (e.g. `check` and `lint`) to validate your WDL files before trying to compile them with dxCompiler.
```
Comment (commandlinegirl) [inline README.md:92]: A note that this limits the usefulness of the `workflow_importer` app that imports workflows from Dockstore.

**Review by commandlinegirl — APPROVED** [review]
Thanks!

**Comment by jdidion** [comment]
the only change is a merge commit, so I am forcing the merge

**Pull request merged**